### PR TITLE
Fix: add missing emojis

### LINKED ISSUE #5371: [QA] Chat | Emojis missing from the panel
### 🛠️ **Build version:**
v0.88.0

### 📔 **Issue Description:**

There are a lot of emojis missing on the build, blank spaces appear on the emojis pannel and there are others that are also missing when trying to find them via the chat.

### 👣 **STR:** 

  1. Open the chat
  2. Start typing` :heart` and check the emojis, search for a plain heart
  3. Open the emojis panel and scroll through them

### ✔️ **Expected Result:**
All emojis are present on the panel, and by typing something like `:heart` all the emojis that contain the word must appear and we must be able to select them.

### ❌ **Actual Result with evidence:**
Some emojis are missing and by typing the word heart it does not appear on the emoji set list

[Player.log](https://github.com/user-attachments/files/22281542/Player.log)

https://github.com/user-attachments/assets/1d114efe-7075-4b6d-b543-b4702a26ead7

### PR DESCRIPTION
**Pull request by lorenzo-ranciaffi**
# Pull Request Description
Fixes #5371 

### Prerequisites
- [ ] List any required setup steps
- [ ] Include environment/configuration requirements

### Test Steps
1. First step
2. Second step
3. Expected result after step 2
4. ...

### Additional Testing Notes
- Note any edge cases to verify
- Mention specific areas that need careful testing
- List known limitations or potential issues

## Quality Checklist
- [ ] Changes have been tested locally
- [ ] Documentation has been updated (if required)
- [ ] Performance impact has been considered
- [ ] For SDK features: Test scene is included

## Code Review Reference
Please review our [Code Review Standards](https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md) before submitting.

### CODE REVIEW

**Review by davidejensen — APPROVED** [review]

File: Explorer/Assets/DCL/UI/InputSuggestions/InputSuggestionPanelController.cs [hunk 1]
```diff
@@ -67,7 +67,8 @@ public Match HandleSuggestionsSearch<T>(string inputText, Regex regex, InputSugg
         private async UniTaskVoid SearchAndSetSuggestionsAsync<T>(string value, InputSuggestionType suggestionType, Dictionary<string, T> suggestionDataMap, CancellationToken ct) where T : IInputSuggestionElementData
         {
             var resultList = new List<T>();
-            await DictionaryUtils.GetKeysWithPrefixAsync(suggestionDataMap, value, resultList, ct);
+            string searchValue = suggestionType == InputSuggestionType.EMOJIS ? value.Replace(":", "") : value;
```
[the remark this answers is not in the document]
Comment (lorenzo-ranciaffi) [reply Explorer/Assets/DCL/UI/InputSuggestions/InputSuggestionPanelController.cs:70]: 🤔 yeah, I guess it will filter for all those ending with `smily:`.
Agree, let's ReplaceAll

**Review by DafGreco — APPROVED** [review]
✔️  PR reviewed and approved by QA on both platforms following instructions playing both happy and un-happy path

Regressions for this ticket had been performed in order to verify that the normal flow is working as expected:

- [✔️ ] Backpack and wearables in world
- [ ✔️] Emotes in world and in backpack
- [✔️ ] Teleport with map/coordinates/Jump In
- [✔️ ] Chat and multiplayer
- [ ✔️] Profile card
- [✔️ ] Camera

Evidence:
image (13)
image (14)
image (12)
